internal/pkg/local/update.go

```diff
@@ -21,7 +21,11 @@ func (m *Manager) UpdatePaths(state model.ObjectState, rename bool) error {
 			v.PathInProject = m.Naming().BranchPath(object.(*model.Branch))
 		case *model.ConfigState:
 			config := object.(*model.Config)
-			v.PathInProject = m.Naming().ConfigPath(v.ParentPath, v.Component, config)
```

Comment:
Povodne obsahoval `ConfigState` aj referenciu na `Component`. 
Je to v tomto code vynimka a anti-pattern, ... kedze vsetky objekty su ulozene v `State`, a kazdy objekt obsahuje iba `key` na ine objekty, ktore sa pomocou `key`, daju vytiahnut zo `state`.

Dovod bol prave tento naming, kde tu `component` potrebuje.
Teraz sa to ale da uz vyriesit spravne, ... kedze sa da vytiahnut komponenta zo `state`.